consensus/polybft/runtime_helpers.go

```diff
@@ -1,5 +1,11 @@
 package polybft
 
+import (
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+type blockGetter func(uint64) (*types.Header, bool)
```

Comment:
You can remove this

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/897/commits/dfc4c8c841fd4738b9946992c15d7b5b031e96a8